README.md

```diff
@@ -90,7 +90,11 @@ Commit the changes, push the branch and open a Pull Request.
 
 ### Import the site to Transition
 
-Once merged, wait for the site to be imported into the Transition app. This currently happens automatically on the hour via a job on deploy.production and deploy.integration Jenkins' during working hours. It can also be triggered manually.
+Once merged, wait for the site to be imported into the Transition app. This currently happens automatically on the hour via a job on deploy.production and deploy.integration Jenkins' during working hours. It can also be triggered manually by running:
+
+[Transition: load site config](https://deploy.integration.publishing.service.gov.uk/job/Transition_load_site_config/)
+
+If the existing site is one that was administered by GDS (e.g. theorytest.direct.gov.uk), you may also need to [update and re-deploy the DNS config](https://docs.publishing.service.gov.uk/manual/dns.html#making-changes-to-publishing-service-gov-uk).
```

Comment:
To be pedantic: this relates to making the site actually transitioned (ie actually redirect), not configuring it. Maybe this is a better fit for the [developer docs](https://docs.publishing.service.gov.uk/manual/transition-a-site.html)?

Reply:
Happy to do that, but where does it fit? Between steps 1 and 2?

Reply:
@issyl0 I've move the DNS stuff: https://github.com/alphagov/govuk-developer-docs/pull/1893